deploy/templates/nstemplatetiers/basic/ns_code.yaml

```diff
@@ -67,11 +67,11 @@ objects:
     limits:
     - type: "Container"
       default:
-        cpu: 1000m
-        memory: 512Mi
+        cpu: 500m
```

Comment:
@alexeykazakov CPU limit should be 1000m, can we apply it across all namespaces?

Reply:
But you did testing for 500m: https://issues.redhat.com/browse/CRT-940?focusedCommentId=15836574&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15836574

Reply:
We could increase it to 1000m though I guess.. The default memory limit allows you to have less than 10 containers anyway. So limiting the number of containers by cpu limits to 10 containers should not change much..
I wish you mentioned that in https://issues.redhat.com/browse/CRT-940?focusedCommentId=15836574&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15836574 before I created the PRs though ;)

Reply:
Frankly, I was focused on RAM only and missed the CPU part during 'test' tier PR review. 
Need to have yet another round of verification on our end, but I suspect that we might need to stick to 750Mb and 1000m limits across namespaces

Reply:
https://github.com/codeready-toolchain/host-operator/pull/380/commits/4af3c3a19232579eec2de2c8d9923a4608bf5db3